### PR TITLE
TEDEFO-367 Made changes listed in ticket, plus some minor corrections.

### PR DESCRIPTION
**Pull request by pdonohoe**
Made changes listed in ticket, plus some minor corrections.

### CODE REVIEW

File: modules/schema/pages/procedure-lot-part-information.adoc [hunk 1]
```diff
@@ -1656,15 +1656,15 @@ shall hold the value associated with that dimension.
 
 The number dimension shall be:
 
-* for weights (BT-5421), elements from the
+* for weights (BT-5421), a value from the
```
Comment (YvesJo) [inline modules/schema/pages/procedure-lot-part-information.adoc:1659]: may be a "code" could be better, "value" could be what is associated to the code

File: modules/schema/pages/procedure-lot-part-information.adoc [hunk 2]
```diff
@@ -1656,15 +1656,15 @@ shall hold the value associated with that dimension.
...
 https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/number-weight[number-weight]
 codelist;
 
-* for fixed values (BT-5422), elements from the
+* for fixed values (BT-5422), a value from the
```
Comment (YvesJo) [inline modules/schema/pages/procedure-lot-part-information.adoc:1663]: same as above

File: modules/schema/pages/procedure-lot-part-information.adoc [hunk 3]
```diff
@@ -1656,15 +1656,15 @@ shall hold the value associated with that dimension.
...
 https://op.europa.eu/web/eu-vocabularies/at-dataset/-/resource/dataset/number-fixed[number-fixed]
 codelist; and
 
-* for threshold values (BT-5423), elements from the
+* for threshold values (BT-5423), a value from the
```
Comment (YvesJo) [inline modules/schema/pages/procedure-lot-part-information.adoc:1667]: same as above

File: modules/schema/pages/procedure-lot-part-information.adoc [hunk 4]
```diff
@@ -2461,7 +2461,7 @@ This term is used to specify that any tenderer must submit tenders for
 all Lots. This element shall be used when and only when tendering for
 all existing Lots is mandatory. For this, the
 "_cbc:PartPresentationCode_" element is used with a single possible
-value: "_all_"
+value: "_all_".
```
Comment (YvesJo) [inline modules/schema/pages/procedure-lot-part-information.adoc:2464]: May be an issue, I think we have "**A**ll" in the codelist

**Review by YvesJo — COMMENTED** [review]
Please let me know whether Doc or codelist should be updated regarding BT-763

File: modules/schema/pages/procedure-lot-part-information.adoc (hunk 1 above)
Comment (pdonohoe) [reply modules/schema/pages/procedure-lot-part-information.adoc]: Changed as suggested

File: modules/schema/pages/procedure-lot-part-information.adoc (hunk 2 above)
Comment (pdonohoe) [reply modules/schema/pages/procedure-lot-part-information.adoc]: Changed as suggested

File: modules/schema/pages/procedure-lot-part-information.adoc (hunk 3 above)
Comment (pdonohoe) [reply modules/schema/pages/procedure-lot-part-information.adoc]: Changed as suggested

File: modules/schema/pages/procedure-lot-part-information.adoc (hunk 4 above)
Comment (pdonohoe) [reply modules/schema/pages/procedure-lot-part-information.adoc:2464]: No the tenderlot-presentation.gc codelist has the lower-case code "all".

**Review by YvesJo — APPROVED** [review]
I didn't see that review request, sorry for that :-S

**Pull request closed without merging**